frontend/projects/upgrade/src/app/core/feature-flags/feature-flags.service.ts

```diff
@@ -17,6 +22,18 @@ export class FeatureFlagsService {
   isLoadingFeatureFlags$ = this.store$.pipe(select(selectIsLoadingFeatureFlags));
   allFeatureFlags$ = this.store$.pipe(select(selectAllFeatureFlagsSortedByDate));
   isAllFlagsFetched$ = this.store$.pipe(select(selectIsAllFlagsFetched));
+  selectSearchString$ = this.store$.pipe(select(selectSearchString));
```

Comment:
can you take the word "select" off of these? That makes them look like the selectors themselves, could be confusing. These streams can just be the name of the value being selected like for other things.